README.md

```diff
@@ -225,7 +255,19 @@ If there are multiple config files, each one of them can be run one after the ot
 *Note: Each step in the profiling pipeline, uses the output from the previous step as its input. Therefore, make sure that all the necessary input files have been generated before running the steps in the profiling pipeline. It is possible to run only a few steps in the pipeline by keeping only those steps in the config file.*
 
 ## Push the profiles to GitHub
-If using a data repository, push the newly created profiles to GitHub as follows
+If using a data repository, push the newly created profiles to DVC and the .dvc files and other files to GitHub as follows
+
+```bash
+dvc add profiles/${BATCH}
```

Comment:
You can send files or folders to DVC but whatever unit you send is tracked as that full unit. So, if you add profiles/BATCH then you can only download that whole folder from DVC. So options for DVC usage are:
- `dvc add` all the profiles in a batch as one unit. This is the simplest and least error prone, but it means that anyone using the profiles will have to download them all as one large unit.
- `dvc add` in more granular units. Makes it a bit more complicated to give explicit instructions, but means that profilers can access just that granular unit.

@shntnu @bethac07 Do you have opinions on whether we should do bulk or granular? (And if granular, how granular?)

Reply:
@shntnu I think `--recursive` is a great solution for both granular and easy :)